### DISCUSSION TITLE
Forced walkers and highways

### DISCUSSION THREAD

**Discussion by tomispev** [Keriew/augustus#742] [General]
I don't use Discord so I'll post this observation here and you can delete it afterwards if it's resolved, or has been already.

Anyway, I've built the Pantheon and I set it so that temples from all over the city send priests to it. Except all the temples that are separated from the Pantheon by a highway don't. In fact the priests flash in front of the temple for a split second. If I delete a section of the highway and place a road there, the priests appear and go to the Pantheon normal. 

Also other forced walkers don't work either. If the southern and northern road to a temple, library, bathhouse, etc. is at some point separated by a highway, the roaming walker will return to where they spawned.

Maybe there are more issues, but I haven't yet come across anything else.

**Comment by PrettyFlower** [comment]
Can you post a saved game with the pantheon issue? I don’t think anyone has reported that yet.

The destination walker thing was deliberate. I didn’t want service workers using highways at all as I didn’t want people trying to use highways to make super loops.

**Reply by tomispev** [reply]
Sure, here it is:
https://github.com/tomispev/game_stuff/blob/main/Augusta.svx

Also maybe you could add it as an option for forced walkers to use highways?

**Reply by PrettyFlower** [reply]
Thx for posting the file! I’ll try to take a look tonight.

We’re trying to avoid adding too many options to the game as it makes the code harder to maintain.

**Reply by PrettyFlower** [reply]
We’ve pushed a fix in to unstable for the pantheon issue, however we’re getting reports of some other bugs in there so you may want to wait a bit before updating.

**Reply by tomispev** [reply]
Thank you! I already updated a few hours ago and it works for the city I've built, so I'll keep playing, but I haven't noticed any bugs yet. Anything to look out for?

**Reply by PrettyFlower** [reply]
There was an issue with drawing gatehouses on highways and another issue with old scenarios not loading. I just pushed some fixes in for both of them.
